rnd/autogpt_server/autogpt_server/schema.prisma

```diff
@@ -0,0 +1,67 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./database.db"
+}
+
+generator client {
+  provider  = "prisma-client-py"
+  interface = "asyncio"
+}
+
+// This model describe a single node in the Agent Graph/Flow (Multi Agent System).
+model AgentNode {
+  id String @id
+
+  // This define the name or the id of the component that this node will execute.
+  // I'm on the fence on whether to make AgentComponent a DB model,
+  // since the list of the components will be hard-coded as the code anyway,
+  // and we won't store code in the DB.
+  //
+  // --- This is how AgentComponent information should look like ---
+  // model AgentComponent {
+  //   id   String @id
+  //   name String
+  //   code String
+  //   type String (Agent/Script)
+  //   inputSchema  String
+  //   outputSchema String 
+  // }
```

Comment:
IMO this is seeded by the system if they don't exist on load. They should be in the DB

Reply:
It's good, to have all in one place though, added :)